src/routes/targeted-messaging/targeted-messaging.controller.spec.ts

```diff
@@ -318,6 +318,79 @@ describe('TargetedMessagingController', () => {
         });
     });
 
+    it('should create a submission for a campaign without targeting enabled', async () => {
+      const outreachId = faker.number.int();
+      const chain = chainBuilder().build();
+      const signerAddress = getAddress(faker.finance.ethereumAddress());
+      const safeAddress = getAddress(faker.finance.ethereumAddress());
+      const submission = submissionBuilder().build();
```

Comment:
Should `submission.build` be `signerAddress` here?

Reply:
https://github.com/safe-global/safe-client-gateway/pull/2194/commits/65896bde7d8fef30ed73e817916f179ddf6cf334#diff-dec390951f4b8861a13a47d81f8e2989f8702c3054dd6562443153fb081f26a7L330